distribution/openrct2.d.ts

```diff
@@ -2495,11 +2495,20 @@ declare global {
          * Highest drop height in height units. Use `context.formatString()` to convert into metres/feet. Ex: `formatString('{HEIGHT}', ride.highestDropHeight)`.
          */
         readonly highestDropHeight: number;
+		
+		/**
+         * Set a breakdown on a ride.
+         * @param breakdown The type of breakdown to set, "none" to fix the ride.
+         */
+        setBreakdown(BreakdownType: breakdown): void;
```

Comment:
I'm not sure I like that we have to set a breakdown to none to fix it, would be probably better to have a separate function like clearBreakdown or fixBreakdown and remove none from BreakdownType as well.

Reply:
Set to none'd.. er- I mean fixed.